test/e2e/specs/nux.test.js

```diff
@@ -2,13 +2,32 @@
  * Internal dependencies
  */
 import '../support/bootstrap';
-import { newDesktopBrowserPage, newPost } from '../support/utils';
+import {
+	clearLocalStorage,
+	clickOnMoreMenuItem,
+	newDesktopBrowserPage,
+	newPost,
+} from '../support/utils';
 
-describe( 'NUX', () => {
-	it( 'should show tips to a first-time user', async () => {
+describe( 'New User Experience (NUX)', () => {
+	const NUMBER_OF_TIPS = 4;
+
+	async function getTipsLocalStorage( page ) {
+		return await page.evaluate( () => {
+			return JSON.parse( window.localStorage.GUTENBERG_NUX_1 );
+		} );
+	}
+
+	beforeEach( async () => {
 		await newDesktopBrowserPage();
 		await newPost( undefined, false );
 
+		// Clear localStorage tips so they aren't persisted for the next test.
```

Comment:
The comment above suggests that it should be part of `afterEach` method. In another test, we do the following:
```js
afterEach( async () => {
		await clearLocalStorage();
		await page.goto( 'about:blank' );
...
```

Reply:
I guess so. It just seems like state is persisted between the tests and I didn't want a previous test messing things up. But I'll try that 👍